<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>motor_control</class>
 <widget class="QMainWindow" name="motor_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>motor_control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>336</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="motor_a_control_buttons">
       <property name="title">
        <string>Motor A control functions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="motor_a_start">
          <property name="text">
           <string>Motor A start</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">motor_a_buttons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="motor_a_stop">
          <property name="text">
           <string>Motor A Stop</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">motor_a_buttons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="motor_b_control_buttons">
       <property name="title">
        <string>Motor B control functions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="motor_b_start">
          <property name="text">
           <string>Motor B start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="motor_b_stop">
          <property name="text">
           <string>Motor B stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="motor_a_buttons"/>
 </buttongroups>
</ui>
